domain/domain.go

```diff
@@ -1046,7 +1046,7 @@ func NewDomain(store kv.Storage, ddlLease time.Duration, statsLease time.Duratio
 		exit:                make(chan struct{}),
 		sysSessionPool:      newSessionPool(capacity, factory),
 		statsLease:          statsLease,
-		infoCache:           infoschema.NewCache(16),
```

Comment:
I don't know if this is a good idea. Because caching cap could easily make OOM, too.

Reply:
I have tested it with 10000 tables, it doesn't consume much memory.

Reply:
How about add a global variable to control it?

Reply:
Or test the 10,000,000 tables?

Reply:
Ok, I don't change the default value now, I add a global variable to control it.